UsersApp/View Controllers/UsersViewController.swift

```diff
@@ -36,30 +43,34 @@ class UsersViewController: UIViewController {
                 DispatchQueue.main.async {
                     self.usersTableView.reloadData()
                 }
-            } catch {
-                print("Error! Can't fetch the users.")
+            } catch let error as NetworkError {
+                print("Network error: \(error.localizedDescription)")
+                let alert = UIAlertController(
+                    title: "Could not fetch the users!",
+                    message: "\(error.localizedDescription)", preferredStyle: .alert)
+                
+                alert.addAction(UIAlertAction(
+                    title: "OK", style: .default, handler: { _ in
+                NSLog("The \"OK\" alert occurred.")
+                }))
```

Comment:
why do you need `NSLog `? u can use `print` to check in consolo.
also, not needed for final PR/review. use `handle: nil `in this case